forte.yml

```diff
@@ -310,7 +310,9 @@ Layer:
           FROM
             osm_places
           WHERE
-            type IN ('state', 'city', 'town') AND geometry && !bbox!
+            name != ''
```

Comment:
![capture d ecran 2018-02-21 a 15 20 35](https://user-images.githubusercontent.com/178744/36484919-0636bd58-171b-11e8-9039-6b280c08878c.png)

Here is one example of a point missing "name" (between Buraydah and Qasim)

Reply:
That one was not that hard to understand ;)
My remark was more about when dealing with font-size and such :)

Reply:
I know, I just tested the functionality of kosmtik, I need to reimport a better country to make the screenshots but was lazy

Reply:
:)